Tests/configure_and_test_integration_instances.py

```diff
@@ -186,6 +189,15 @@ class Build(ABC):
     DEFAULT_SERVER_VERSION = '99.99.98'
 
     #  END CHANGE ON LOCAL RUN  #
+    @staticmethod
+    def get_secret_service_account_config() -> str:
+        print('##################################')
+        print('##################################')
+        print('##################################')
+        print(os.environ['GSM_SERVICE_ACCOUNT'])
+        with open(os.environ['GSM_SERVICE_ACCOUNT']) as f:
+            creds = json.load(f)
+            return json.dumps(creds)
```

Comment:
follow this functions and replace the secrets accordingly
<img width="746" alt="image" src="https://user-images.githubusercontent.com/7270217/204127229-30e533a4-2229-4119-9b4d-449fe56e4ee2.png">
configure_and_install_packs.py